app/decorators/document_decorator.rb

```diff
@@ -0,0 +1,17 @@
+class DocumentDecorator < Draper::Decorator
+  delegate_all
+
+  def name_link
```

Comment:
what do you think about renaming those methods ..
`name_link` => `document_page_link` (or `just document_link`)
`open_link` => `document_url_link`
.. to indicate where those links actually go.